packages/core/admin/server/services/metrics.js

```diff
@@ -20,8 +20,28 @@ const sendDidChangeInterfaceLanguage = async () => {
   strapi.telemetry.send('didChangeInterfaceLanguage', { userProperties: { languagesInUse } });
 };
 
+const sendUpdateProjectInformation = async () => {
+  const numberOfActiveAdminUsers = await getService('user').count({ isActive: true });
+  const numberOfAdminUsers = await getService('user').count();
+
+  strapi.telemetry.send('didUpdateProjectInformation', {
+    groupProperties: { numberOfActiveAdminUsers, numberOfAdminUsers },
+  });
+}
+
+const startCron = (strapi) => {
+  strapi.cron.add({
+    '0 0 0 * * *' :() => {
+      sendUpdateProjectInformation();
+    }
```

Comment:
I think the lint is complaining about this bit needing to be a shorthand method
```suggestion
    '0 0 0 * * *': () => sendUpdateProjectInformation()
```